AK/String.cpp

```diff
@@ -309,6 +311,70 @@ ErrorOr<String> String::replace(StringView needle, StringView replacement, Repla
     return StringUtils::replace(*this, needle, replacement, replace_mode);
 }
 
+bool String::contains(String const& needle, CaseSensitivity case_sensitivity) const
```

Comment:
not convinced we need this overload, just taking a string view in all cases seems fine?

Reply:
fixed